docs/faq/Others.md

```diff
@@ -188,7 +188,14 @@ Run the `select current_version();` command or the CLI command `./bin/show_fe_ve
 
 ## How to set the memory size of an FE?
 
-FEs are used to store metadata. You can set the memory size of an FE based on the number of tablets. In StarRocks, you can set the memory size of an FE to 20 GB at most. 10 million tablets occupy about 20 GB of an FE memory.
+The metadata is stored in the memory used by the FE. You can set the memory size of the FE according to the number of tablets as shown in the diagram below. For example, if the number of tablets is below 1 million, you should allocate a minimum of 16GB of memory to the FE. You can configure Xms and Xmx values at the **JAVA_OPTS=** in the **fe.conf** file, and the Xms and Xmx values should be consistent. Additionally, the configuration should be same across all FEs because each FE can be a Leader.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/740b69e4-54f4-4cb2-917d-973d3e17301c)
-> any of the FEs can be elected as a Leader